addons/sourcemod/scripting/zr_personal_skins.sp

```diff
@@ -135,346 +158,247 @@ public void CvarChanges(ConVar convar, const char[] oldValue, const char[] newVa
-				if(ZR_IsClientHuman(client) && g_sPlayerModelHuman[client][0])
-				{
-					newMode = CHANGE_MODE_CHANGESKIN_TO_PSKIN;
-					SetEntityModel(client, g_sPlayerModelHuman[client]);
-				}
-			}
-			
-			g_iClientPreviousChangeMode[client] = CHANGE_MODE_CHANGED;
-			g_iClientChangeMode[client] = newMode;
-			g_bClientChangedSkin[client] = true;
-		}
-		else
-		{
-			int modeEx = g_iClientChangeMode[client];
-			int newMode;
-		
-			if(!g_bClientDisabledSkin[client])
-				newMode = CHANGE_MODE_CHANGESKIN_TO_PSKIN;
-			else
-				newMode = CHANGE_MODE_CHANGESKIN_TO_ACTUAL;
-		
-			g_iClientPreviousChangeMode[client] = modeEx;
-			g_iClientChangeMode[client] = newMode;
-			CReplyToCommand(client, "{green}[ZR] {default}Your changes will be applied on next round!");
-		}
-		
+	if (!client)
 		return Plugin_Handled;
-	}
-	else
-	{
-		int mode = g_iClientChangeMode[client];
-		int newMode;
-		
-		if(!g_bClientDisabledSkin[client])
-			newMode = CHANGE_MODE_CHANGESKIN_TO_PSKIN;
-		else
-			newMode = CHANGE_MODE_CHANGESKIN_TO_ACTUAL;
-		
-		g_iClientPreviousChangeMode[client] = mode;
-		g_iClientChangeMode[client] = newMode;
-		CReplyToCommand(client, "{green}[ZR] {default}Your changes will be applied on next round!");
-		return Plugin_Handled;
-	}
-}
 
-public Action Command_TogglePSkin(int client, int args)
-{
-	if(args < 2)
-	{
-		CReplyToCommand(client, "{green}[ZR] {default}Usage: sm_togglepskin <player> <1|0>");
-		return Plugin_Handled;
-	}
-	
-	char arg1[65], arg2[7];
-	GetCmdArg(1, arg1, sizeof(arg1));
-	GetCmdArg(2, arg2, sizeof(arg2));
-	
-	int target = FindTarget(client, arg1, true, false);
-	if(target < 1)
-	{
-		CReplyToCommand(client, "{green}[ZR] {default}Invalid target.");
-		return Plugin_Handled;
-	}
-	
-	if(!g_bIsPlayerHasSkins[target])
-	{
-		CReplyToCommand(client, "{green}[ZR] {default}The specified target doesn't have a personal skin!");
-		return Plugin_Handled;
-	}
-	
-	int num;
-	if(!StringToIntEx(arg2, num))
-	{
-		CReplyToCommand(client, "{green}[ZR] {default}Invalid toggle value.");
-		return Plugin_Handled;
-	}
-	
-	bool bValue;
-	if(num <= 0)
-		bValue = true;
+	if (!g_bHasPersonalSkinsZombie[client] && !g_bHasPersonalSkinsHuman[client])
+		CReplyToCommand(client, "{green}[ZR] {default}You don't have personal-skin");
 	else
-		bValue = false;
-		
-	if(g_bForceDisableSkin[target] == bValue)
-	{
-		CReplyToCommand(client, "{green}[ZR] {default}The specified target's personal skin is already %s.", (bValue) ? "{red}disabled" : "{green}enabled");
-		return Plugin_Handled;
-	}
-	
-	g_bForceDisableSkin[target] = bValue;
-	g_hForced.Set(client, "true");
-	CReplyToCommand(client, "{green}[ZR] {red}Successfully %s {default}Personal Skin on {olive}%N.", (bValue) ? "{red}disabled" : "{green}enabled", target);
-	LogAction(client, target, "[ZR-PersonalSkin] \"%L\" Force-%s \"%L\"'s personal skin", client, (bValue) ? "Disabled" : "Enabled", target);
+		FakeClientCommand(client, "zclass"); // Print zclass menu to client, let him choose
```

Comment:
not urgent but create an issue to avoid fakeclientcommand and instead call the real zr function that displays zclass

Reply:
ive opened it here:
- https://github.com/srcdslab/sm-plugin-ZR_PersonalSkins/issues/8